### PR TITLE
LPS-148013 Move petra-json-web-service-client to portal-json-web-service-client

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina,

Main Task: [LPS-147928](https://issues.liferay.com/browse/LPS-147928)
SubTask: [LPS-148013](https://issues.liferay.com/browse/LPS-148013)

### CODE REVIEW

**Comment by julschong** [comment]
@catarineaguiar Hi Catarine,
could you please help me scan the test result?
Thank you!

**Comment by catarineaguiar** [comment]
@julschong no regressions found.

**Comment by julschong** [comment]
Hi Tina, @tinatian
could you please help review this PR?
Thank you!

File: modules/dxp/apps/commerce/commerce-machine-learning-impl/build.gradle [hunk 1]
```diff
@@ -25,11 +25,11 @@ dependencies {
 	compileOnly project(":apps:dispatch:dispatch-api")
 	compileOnly project(":apps:frontend-taglib:frontend-taglib")
 	compileOnly project(":apps:info:info-api")
-	compileOnly project(":apps:petra:petra-json-web-service-client")
 	compileOnly project(":apps:portal-search:portal-search-api")
 	compileOnly project(":apps:portal-search:portal-search-engine-adapter-api")
 	compileOnly project(":apps:portal-vulcan:portal-vulcan-api")
 	compileOnly project(":apps:portal:portal-instance-lifecycle-api")
+	compileOnly project(":apps:portal:portal-json-web-service-client")
```
Comment (tinatian) [inline modules/dxp/apps/commerce/commerce-machine-learning-impl/build.gradle:32]: @julschong No direct reference to the packages exported by portal-json-web-service-client, we should remove this compileOnly.

Tina.

Comment (julschong) [reply modules/dxp/apps/commerce/commerce-machine-learning-impl/build.gradle]: pushed again with it removed

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test

**Comment by julschong** [comment]
forced pushed to remove unused dependency

**Comment by julschong** [comment]
going to rebase and submit again

**Pull request closed without merging**